xknx/telegram/apci.py

```diff
@@ -972,6 +979,73 @@ def __str__(self) -> str:
         return "<Restart />"
 
 
+class ConfirmedRestart(APCI):
+    """Restart service."""
+
+    CODE = APCIService.CONFIRMED_RESTART
+
+    def __init__(self, erase_code: bytes, channel: bytes) -> None:
+        """Initialize a new instance of FunctionPropertyCommand."""
+        self.erase_code = erase_code
+        self.channel = channel
+
+    def calculated_length(self) -> int:
+        """Get length of APCI payload."""
+        return 3
+
+    @classmethod
+    def from_knx(cls, raw: bytes) -> ConfirmedRestart:
+        """Parse/deserialize from KNX/IP raw data."""
+        erasecode, channel = struct.unpack("!cc", raw[2:])
+        return cls(erase_code=erasecode, channel=channel)
+
+    def to_knx(self) -> bytearray:
+        """Serialize to KNX/IP raw data."""
+        payload = struct.pack("!cc", self.erase_code, self.channel)
+        return encode_cmd_and_payload(self.CODE, appended_payload=payload)
+
+    def __str__(self) -> str:
+        """Return object as readable string."""
+        return "<Confirmed Restart />"
+
+
+class RestartResponse(APCI):
+    """
+    RestartResponse service.
+
+    Payload contains the Errorcode and DPTTimePeriodSec.
+    """
+
+    CODE = APCIService.CONFIRMED_RESTART_RESPONSE
+
+    def __init__(self, error_code: bytes, time: bytes) -> None:
```

Comment:
Consider using `int` or `enum` for error_code and `int` for time. Maybe rename time to `time_sec` or `process_time` (like in the specs).